### PR TITLE
Add inline docs to Assertions interface in type definitions

### PR DESCRIPTION
**Pull request by malimichael**
I believe it is valuable to have inline docs here. If I type `t.` in my editor I like to be able to scroll through all the assertion options and see what they do.

### CODE REVIEW

**Comment by novemberborn** [comment]
Thanks for the PR @malimichael. Can I ask what editor you're using? I do see the behavior you're describing in Atom.

Could you make the same changes in `index.js.flow`? We're trying to keep these files as similar as possible.

**Comment by malimichael** [comment]
@novemberborn I am using VS Code v 1.22.1

**Comment by malimichael** [comment]
This is what Im getting before changes for all assertions before changes

screenshot 2018-04-07 19 40 02


And this afterwards 

screenshot 2018-04-07 19 38 41

File: index.js.flow [hunk 1]
```diff
@@ -31,20 +31,61 @@ export type SnapshotOptions = {
...
+	/**
+	 * Assert that `actual` is [the same
+	 * value](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is) as `expected`.
+	 */
 	is: IsAssertion;
+
+	/**
+	 * Assert that `actual` is not [the same
+	 * value](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is) as `expected`.
+	*/
```
Comment (sindresorhus) [inline index.js.flow]: This last `*/` is not aligned.

Comment (malimichael) [reply index.js.flow]: 😅

**Comment by novemberborn** [comment]
I caught a few more whitespace fixes, but it's all good now.

Thank you @malimichael!

**Pull request merged**